zerver/tests/test_subs.py

```diff
@@ -2017,7 +2031,8 @@ def test_inviteonly(self):
         self.assertEqual(json["already_subscribed"], {})
 
         # Make sure both users are subscribed to this stream
-        result = self.client_get("/api/v1/streams/%s/members" % (stream_name,),
+        stream_id = Stream.objects.get(name=stream_name).id
```

Comment:
could we open a followup issue for this ? because there are also `Stream.objects.get()` that not on my change